src/components/helpers.component.ts

```diff
@@ -74,6 +74,14 @@ export class ContextAwareHelpersComponent extends ContextAwareRendererComponent
     Handlebars.registerHelper('breadcrumbs', function(this: PageEvent) {
       return component.breadcrumb(this.model, this.project, []);
     });
+
+    Handlebars.registerHelper('if_eq', function(a: any, b: any, opts: any) {
```

Comment:
Any pure helper functions should go in 'src/resources/helpers'. However I think you can use `ifCond` helper from typedoc core https://github.com/TypeStrong/typedoc/blob/master/src/lib/output/helpers/if-cond.ts as this will already be registered to the Handlebars instance.